src/styles/components/MasterHeader.scss

```diff
@@ -53,6 +54,16 @@ nav.navbar{
   }
   .btn-link{
     height: 40px;
+    padding-left: 0;
+    padding-right: 0;
+    width: 100%;
+    text-align: left;
+
+    .caret {
+      position: absolute;
+      right: 12px;
+      top: 19px;
```

Comment:
what browsers do we care about at what min version (just curious)

On Thu, Apr 7, 2016 at 8:26 AM Mohamed El Mahallawy <
notifications@github.com> wrote:

> In src/styles/components/MasterHeader.scss
> <https://github.com/quran/quran.com-frontend/pull/258#discussion_r58824144>
> :
>
> > @@ -53,6 +54,16 @@ nav.navbar{
> >    }
> >    .btn-link{
> >      height: 40px;
> > +    padding-left: 0;
> > +    padding-right: 0;
> > +    width: 100%;
> > +    text-align: left;
> > +
> > +    .caret {
> > +      position: absolute;
> > +      right: 12px;
> > +      top: 19px;
>
> http://caniuse.com/#search=flex
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/quran/quran.com-frontend/pull/258/files/e00db328e72a70a6f672982312e611272f2e6d0a#r58824144>
>